test/libsolidity/semanticTests/types/mapping_enum_key_library_v2.sol

```diff
@@ -1,4 +1,4 @@
-pragma experimental ABIEncoderV2;
+pragma abicoder v2;
```

Comment:
I think we should remove most of these, as it is the default, but would do it in a specific PR dealing with that only as it is around 200 locations.

Reply:
After reviewing #10643 and seeing how the `--abiencoderv1` option really works I see that v2 pragma and the lack of pragma are not actually equivalent. If the pragma is there, `--abiencoderv1` won't try switching the test to v1 by adding the v1 pragma but it will if the pragma is missing.

So I still think this is a good idea but you'll need to run it with `--abiencoderv1` and possibly re-add some of the pragmas in tests that are v2-only.

Reply:
I haven't removed any v2 pragma.

Reply:
Yeah, I meant re-adding one by one just after you remove them all.